Makefile

```diff
@@ -60,7 +60,7 @@ LD_FLAGS ?= "-X $(VERSION_PKG).version=$(VERSION) -X $(VERSION_PKG).buildDate=$(
 # ENVTEST_K8S_VERSION refers to the version of kubebuilder assets to be downloaded by envtest binary.
 ENVTEST_K8S_VERSION = 1.22
 # Options for kuttl testing
-KUBE_VERSION ?= 1.20
+export KUBE_VERSION ?= 1.20
```

Comment:
why do we need this export?

Reply:
Because doing this, we can use `KUBE_VERSION` as environment variable of the scripts called by `make`